.circleci/config.yml

```diff
@@ -191,35 +191,40 @@ jobs:
     steps:
       - run: echo "hello from branch ${CIRCLE_BRANCH}"
 workflows:
+  ### on bors action or PR ###
   build-test-deploy:
+    when:
+      condition:
+        or:
+        - equal: [ auto, << pipeline.git.branch >> ]
+        - equal: [ canary, << pipeline.git.branch >> ]
+        - matches:
+            pattern: "pull/.*"
```

Comment:
doesn't this map to everyone? or is your intent to block that with the ensuing PR?

Reply:
Yea the workflow is triggered either by bors action, or by any PR made to the repo. We  can further scope it down with CircleCI contexts  gated via GH groups, but these are the main conditions

Reply:
that's my concern is that this PR doesn't seem to actually scope it down yet? or is that done in circleci's website?